tutorials/image-curation/image-curation.ipynb

```diff
@@ -0,0 +1,528 @@
+    "id_col = \"key\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_curator.datasets import ImageTextPairDataset\n",
+    "\n",
+    "dataset = ImageTextPairDataset.from_webdataset(dataset_path, id_col)\n",
+    "# Filter out any entries that failed to download\n",
+    "dataset.metadata = dataset.metadata[dataset.metadata[\"error_message\"].isna()]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Choose the Embedder\n",
+    "We can now define the embedding creation step in our pipeline. NeMo Curator has support for all [timm](https://pypi.org/project/timm/) models. NeMo Curator's aesthetic classifier is trained on embeddings from `vit_large_patch14_clip_quickgelu_224.openai`, so we will use that.\n",
+    "\n",
+    "The cell below will do the following:\n",
+    "1. Download the model `vit_large_patch14_clip_quickgelu_224.openai`.\n",
+    "1. Automatically convert the image preprocessing transformations of `vit_large_patch14_clip_quickgelu_224.openai` from their PyTorch form to DALI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_curator.image.embedders import TimmImageEmbedder\n",
+    "\n",
+    "embedding_model = TimmImageEmbedder(\n",
+    "                    \"vit_large_patch14_clip_quickgelu_224.openai\",\n",
+    "                    pretrained=True,\n",
+    "                    batch_size=1024,\n",
+    "                    num_threads_per_worker=16,\n",
+    "                    normalize_embeddings=True,\n",
+    "                    autocast=False,\n",
+    "                )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now create embeddings for the whole dataset. It's important to understand what is going on internally in NeMo Curator so you can modify parameters appropriately.\n",
+    "\n",
+    "Once the computation is triggered, the cell below will\n",
+    "1. Load a shard of metadata (a `.parquet` file) onto each GPU you have available using Dask-cuDF.\n",
+    "1. Load a copy of `vit_large_patch14_clip_quickgelu_224.openai` onto each GPU.\n",
+    "1. Repeatedly load images into batches of size `batch_size` onto each GPU with a given threads per worker (`num_threads_per_worker`) using DALI.\n",
+    "1. The model is run on the batch (without `torch.autocast()` since `autocast=False`).\n",
+    "1. The output embeddings of the model are normalized since `normalize_embeddings=True`.\n",
+    "\n",
+    "\n",
+    "Since NeMo Curator uses Dask, the cell below will not cause the embeddings to be created. The computation will only begin once we inspect the output in the `dataset.metadata.head()` call or when we write to disk using `dataset.save_metadata()` or `dataset.to_webdataset()`."
```

Comment:
Maybe elaborate a bit more on when to use `save_metadata` versus `to_webdataset`?